bin/test_integration_rhel

```diff
@@ -53,7 +89,9 @@ function _configure_conjur() {
   _cmd="$_cmd --hostname $TEST_HOSTNAME"
   _cmd="$_cmd --master-altnames $TEST_HOSTNAME"
   _cmd="$_cmd --admin-password $ADMIN_PASSWORD"
-  _cmd="$_cmd dev"
+  _cmd="$_cmd ${ACCOUNT}"
```

Comment:
should `ACCOUNT` be set to `test` instead?

Reply:
@orenbm I think it should be set form the docker-compose we just ignored it

Reply:
i don't see it set in `docker-compose` but in line 68 of this file. i would change it there.

Reply:
it is set in test_integration_rhel you can override it in docker-compose the change I did was not to ignore the ACCOUNT var